### PR TITLE
 #535917 added styles for sxa initializer(basic site)

### PR DESCRIPTION
**Pull request by matkovskyi**
added additional styles for basic site(sxa initializer)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx [hunk 1]
```diff
@@ -65,11 +65,11 @@ export const WithText = (props: PromoProps): JSX.Element => {
           <div className="promo-text">
             <div>
               <div className="field-promotext">
-                <Text className="promo-text" field={props.fields.PromoText} />
+                <Text className="promo-text" field={props.fields.PromoText} encode={false} />
```
Comment (illiakovalenko) [inline packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx:68]: What is the reason do disable encoding?

**Review by illiakovalenko — CHANGES_REQUESTED** [review]
Can you add End of line for some files? Check them

File: packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx (hunk 1 above)
Comment (matkovskyi) [reply packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx:68]: I reverted and changed Text component to RichText. In PromoText we use html-text

File: packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const Default = (props: PromoProps): JSX.Element => {
           <div className="promo-text">
             <div>
               <div className="field-promotext">
-                <Text className="image-caption" field={props.fields.PromoText} />
+                <JssRichText field={props.fields.PromoText} />
```
[the remark this answers is not in the document]
Comment (matkovskyi) [reply packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx:43]: Yes, my bad. fixed

**Review by illiakovalenko — APPROVED** [review]

**Pull request merged**